CMakeLists.txt

```diff
@@ -287,6 +287,15 @@ if(WIN32)
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_CRT_SECURE_NO_WARNINGS")
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DWIN32_LEAN_AND_MEAN")
 
+	set(CMAKE_USE_RELATIVE_PATH ON)
```

Comment:
I wouldn't generally change the cmake rpath behavior if not absolutely required. At least maybe this should only be done on windows?